spec/controllers/rents_controller_spec.rb

```diff
@@ -31,6 +31,19 @@
           expect(JSON.parse(response.body)['total_count']).to eq(0)
         end
       end
+
+      context 'Without user.id' do
+        it 'returns forbidden error' do # rubocop:disable ExampleLength
+          # I need to open class becouse an user.id never must be nil
+          class RentPolicy < ApplicationPolicy
```

Comment:
Remember what we talked about declaring methods in spec files, the same applies to classes. The cases where you have to do something like this are very limited, and this isn't one of those cases!
Some cases you could test here:
- Try to access to a record belonging to another user, that should return a :forbidden response
- Try to access to a record belonging to the logged in user, that should return that record serialized.